lib/web_ui/skwasm/library_skwasm_support.js

```diff
@@ -0,0 +1,81 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+mergeInto(LibraryManager.library, {
```

Comment:
What sorcery is this? Is this Emscripten? Maybe leave a comment with some breadcrumbs?

Reply:
Yes it's the emscripten JS library magic. This JS file is preprocessed by emscripten and folded into `skwasm.js`. I will leave some comments.